spec/lib/wiki_pageviews_spec.rb

```diff
@@ -47,14 +47,12 @@
 
       context 'beyond the allowed date range' do
         let(:start_date) { '2015-01-01'.to_date }
+        let(:end_date) { '2015-01-02'.to_date }
 
         it 'does not raise an error' do
-          stub_request(:any, /.*wikimedia.org.*/)
-            .to_return(
-              status: 404,
-              body: '{"type":"https://mediawiki.org/wiki/HyperSwitch/errors/not_found"}'
-            )
-          expect { subject }.not_to raise_error
+          VCR.use_cassette 'wiki_pageviews/views_for_article' do
+            expect { subject }.not_to raise_error
```

Comment:
I don't think that stubbing the request makes sense now. Instead of that, I set start and end date out of the "allowed data range"